src/site/content/en/fast/measure-performance-with-the-rail-model/index.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: Measure performance with the RAIL model
+authors:
+  - addyosmani
+  - developit
+  - kaycebasques
+  - megginkearney
+description: |
+  RAIL model enables designers and developers to reliably target the performance optimization work that has the highest impact on user experience. Learn what goals and guidelines the RAIL model sets out and which tools you can use to achieve them.
+date: 2020-06-05
+---
```

Comment:
```suggestion
date: 2020-06-05
tags:
  - performance
  - animations
  - devtools
  - lighthouse
  - metrics
  - mobile
  - network
  - rendering
  - ux
---
```
I can't remember if tags work outside of the blog but it's worth a try

Reply:
yeah i believe they'll appear at the bottom of the post